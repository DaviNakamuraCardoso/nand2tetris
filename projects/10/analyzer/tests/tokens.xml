<keyword>class</keyword><identifier>Hello</identifier><symbol>{</symbol><keyword>function</keyword><keyword>void</keyword><identifier>hello</identifier><symbol>(</symbol><symbol>)</symbol><symbol>{</symbol><identifier>Output</identifier><symbol>.</symbol><identifier>printString</identifier><symbol>(</symbol><stringConstant>Hello!</stringConstant><symbol>)</symbol><symbol>}</symbol><symbol>}</symbol>