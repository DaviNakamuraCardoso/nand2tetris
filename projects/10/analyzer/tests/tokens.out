class
Apalachicola
{
constructor
new
(
)
{
do
Output
.
printString
(
"We are on our way."
)
;
do
Output
.
printString
(
"==>> to Apalachicola F.L.A"
)
;
do
Output
.
println
(
)
;
return
;
}
}
